src/run.ts

```diff
@@ -114,3 +115,42 @@ async function runSetup(): Promise<null | Error> {
   );
   return null;
 }
+
+function resolveVersionInput(): string {
+  const version = core.getInput("version");
+  const versionFilePath = core.getInput("go_version_file");
+
+  if (versionFilePath !== "") {
+    if (version !== "") {
+      core.warning(
+        "Both version and go_version_file inputs are specified, go_version_file will be preferred"
```

Comment:
We would definitely prefer the version in this case

Reply:
Unfortunately that's not possible as version has a default.